rtf_vis_tool/src/result_metrics/multiview_optimizer_metrics.json

```diff
@@ -0,0 +1,113 @@
+{
+    "rotation_averaging_angle_deg": {
+        "median_error": 2.915834376502234,
```

Comment:
this looks like sample data, that probably belongs in a unit test. Is there a way to use them in a unit test?

Reply:
will remove all the files in result_metrics since those are generated anyways after running scene_optimizer.py. Will look more into unit testing.